### ISSUE TITLE
Filter table by non-int foreign key results in error 'Could not interpret XXX as an integer'

### ISSUE THREAD

**Issue by danielbotella** [iommirocks/iommi#523]
When filter a table on a foreign-key column where the foreign key is not an integer, selecting a value from the choice dropdown results in an error: 'Could not interpret XXX as an integer'.

**Comment by jlubcke** [comment]
I see no obvious reason for why we should not just let the pk value through to django even if it is not an integer.

**Comment by boxed** [comment]
Agreed. I think we can just delete that code. I'm looking at it right now.

**Comment by jlubcke** [comment]
Pr: #524

**Comment by jlubcke** [comment]
Also: This looks like a duplicate: #493

**Comment by boxed** [comment]
This is fixed in 6.8.0.
